Brain/ChatBot.py

```
@@ -28,11 +30,36 @@ def __init__(self, query=None):
             with open(self.db_file, 'w', encoding='utf-8') as f:
                 json.dump([], f, ensure_ascii=False, indent=4)
 
-        # 4) Handle the query
-        self.query = query
-
-
-        print(f"🗣️ User: {self.query}")
+    async def handle_query(self, query, contexts:list = ["General"]):
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)